src/helpers.ts

```diff
@@ -0,0 +1,20 @@
+import type { HonoRequest } from "hono";
+import type z from "zod";
+
+export async function requestBody<T extends z.ZodType = z.ZodTypeAny>(
```

Comment:
This method can now be used to clean up all the cases where we've request data coming from both the FormData and JSON.